tardis/montecarlo/setup_package.py

```diff
@@ -30,14 +31,16 @@ def get_extensions():
     deps += [os.path.relpath(fname) for fname in glob(
         os.path.join(os.path.dirname(__file__), 'src/randomkit', '*.h'))]
 
-    return [Extension('tardis.montecarlo.montecarlo', sources,
-                      include_dirs=['tardis/montecarlo/src',
-                                    'tardis/montecarlo/src/randomkit',
-                                    np.get_include()],
-                      depends=deps,
-                      extra_compile_args=compile_args,
-                      extra_link_args=link_args,
-                      define_macros=define_macros)]
+    return cythonize(
+        Extension('tardis.montecarlo.montecarlo', sources,
+            include_dirs=['tardis/montecarlo/src',
+                'tardis/montecarlo/src/randomkit',
+                np.get_include()],
```

Comment:
This should use `include_dirs=['numpy']` as documented here: http://docs.astropy.org/en/latest/development/ccython.html#using-numpy-c-headers